e2e_playwright/st_expander.py

```diff
@@ -43,3 +43,27 @@ def update_value():
 
 if st.button("Print State Value"):
     st.text(st.session_state.number)
+
+expander_long = st.expander(
+    "Expand me! "
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum arcu nisl, tincidunt id "
+    "orci id, condimentum cursus nunc. Nullam sed sodales ipsum, vel tincidunt dui. Etiam diam "
+    "dolor, eleifend sit amet purus id, dictum aliquam quam.",
+    expanded=True
```

Comment:
This fixes the lint error seen in [`js-test` CI job](https://github.com/streamlit/streamlit/actions/runs/7315429829/job/20001194677?pr=7596):
```suggestion
    expanded=True,
```